src/vehicle_use.cpp

```diff
@@ -55,6 +55,7 @@
 #include "vpart_position.h"
 #include "vpart_range.h"
 #include "weather.h"
+#include "handle_action.cpp"
```

Comment:
And include `handle_action.h` here instead.

Reply:
Thanks!